packages/ui/src/components/Document/actions/DetachSchemaButton.tsx

```diff
@@ -26,9 +26,14 @@ import { DocumentService } from '../../../services/document.service';
 type DeleteSchemaProps = {
   documentType: DocumentType;
   documentId: string;
+  parameterReferenceId: string;
 };
 
-export const DetachSchemaButton: FunctionComponent<DeleteSchemaProps> = ({ documentType, documentId }) => {
+export const DetachSchemaButton: FunctionComponent<DeleteSchemaProps> = ({
+  documentType,
+  documentId,
+  parameterReferenceId,
```

Comment:
Oops overlooked this, this would be better to be `documentReferenceId`, as `DetachSchemaButton` is also used for source body and target body, so the common name is a document.